smarts/core/vehicle.py

```diff
@@ -526,6 +527,16 @@ def attach_sensors_to_vehicle(sim, vehicle, agent_interface, plan):
             )
         )
 
+        if agent_interface.traffic_lights:
+            assert (
+                sim.scenario.traffic_history
+            ), "Traffic history required for traffic lights sensor"
+            vehicle.attach_traffic_lights_sensor(
+                TrafficLightSensor(
+                    sim.scenario.traffic_history, sim.scenario.road_map.xy_offset
```

Comment:
Of course, when this gets merged with PR #1075, this will need to change.  Just flagging it so we don't miss it.